test/smoke/MLH-543_storytoRead_TCS.js

```diff
@@ -6,34 +6,36 @@ const inputValues4AndClick = require('../../helpers/inputValues4AndClick');
 
 describe('MLH-543 Page header is present and correct  ', function() {
 
-    it('TCS-001 page header My little hero is present', function (){
+    xit('TCS-001 page header My little hero is present', function (){
         browser.url('');
         const headerMLH = $(sel.headerMLH).isDisplayed();
         expect(headerMLH).toEqual(true);
     });
 
-    it('TCS-002 page header My little hero is correct', function (){
+    xit('TCS-002 page header My little hero is correct', function (){
         const headerMLH = $(sel.headerMLH).getText();
         expect(headerMLH).toEqual(exp.headerMLH);
     });
 //TCS-003 unresolved yet
-    it('TCS-004 uploaded image has size <=500px', function (){
+    xit('TCS-004 uploaded image has size <=500px', function (){
         inputValues5AndClick(data.name,data.gender.she,data.age,data.storyType.Comedy);
         const uploadedPictureSize = $(sel.uploadedPicture).getAttribute('width');
         const toNumber = parseInt(uploadedPictureSize, 10);
         expect(toNumber).toBeLessThanOrEqual(500);
     });
 
-    it('TCS-005 have default image on the story', function (){
+    xit('TCS-005 have default image on the story', function (){
         browser.url('');
         inputValues4AndClick(data.name,data.gender.she,data.age,data.storyType.Comedy);
         const uploadedPicture = $(sel.uploadedPicture).getAttribute('src');
         expect(uploadedPicture).toContain('app/static');
     });
 
     it('TCS-006 the header of the story is in capital letters', function (){
-
-
+        browser.url('');
+        inputValues4AndClick(data.name,data.gender.she,data.age,data.storyType.Comedy);
+        const headerStory = $(sel.headerStory).getText();
+        expect(headerStory).toStrictEqual(data.headerStory);
```

Comment:
If you are checking capital letters then why using toStrictEqual? If you're checking if the Name of the story is correct then toStrictEqual is OK but it gives a positive test while the name of the story is wrong "Two Cats And A Name 123"

Reply:
it does not mater - we check capital letters and not capital letters + 123. Perfectly works with  two Cats And A Name as well as other variants